.github/scripts/findUnusedStyles.ts

```diff
@@ -0,0 +1,561 @@
+import * as fs from 'fs';
+import {globSync} from 'glob';
+import * as path from 'path';
+import * as ts from 'typescript';
+import TSCompilerUtils from '../../scripts/utils/TSCompilerUtils';
+
+type StyleDefinition = {
+    key: string;
+    file: string;
+    line: number;
+    column: number;
+};
+
+// Static patterns and constants - created once
+const STYLE_FILE_EXTENSIONS = '**/*.{ts,tsx,js,jsx}';
```

Comment:
Do we still have styles in `.js` or `.jsx` files? I don't think so?

Reply:
I can remove it, was just in case